planet/scripts/cli.py

```diff
@@ -103,3 +107,37 @@ def init(email, password):
     response = call_and_wrap(clientv1().login, email, password)
     write_planet_json({'key': response['api_key']})
     click.echo('initialized')
+
+@cli.command('quota')
+def quota():
+    '''Print allocation and remaining quota in Sqkm.'''
+    try:
+        with open (os.path.join(os.path.expanduser('~'), '.planet.json')) as f:
+            data=json.load(f)
```

Comment:
`f` here is the same as `planet.api.utils._planet_json_file()` -- however you can simplify this file read entirely, and avoid code duplication, by calling `planet.api.utils.read_planet_json()` (see: https://github.com/samapriya/planet-client-python/blob/master/planet/api/utils.py#L31)